docs/source/concepts/ComponentInfo.rst

```diff
@@ -9,7 +9,28 @@ ComponentInfo
 
 Introduction
 ------------
-``ComponentInfo`` provides faster and simpler access to instrument/beamline geometry as required by Mantid :ref:`Algorithms <Algorithm>` than was possible using :ref:`Instrument`. ``ComponentInfo`` and :ref:`DetectorInfo` are designed as full replacements to :ref:`Instrument`. At the time of writing, `ComponentInfo` is incomplete.
+``ComponentInfo`` provides faster and simpler access to instrument/beamline geometry as required by Mantid :ref:`Algorithms <Algorithm>` than was possible using :ref:`Instrument`. ``ComponentInfo`` and :ref:`DetectorInfo` are designed as full replacements to :ref:`Instrument`. 
 
-Until complete, :ref:`Instrument Access Layers <InstrumentAccessLayers>` provides the best details on how `ComponentInfo` can be used in it's current state of implementation.
+:ref:`Instrument Access Layers <InstrumentAccessLayers>` provides details on how `DetectorInfo` interacts with other geometry access layers.
+
+Python Interface
+----------------
+
+Examples of using ``ComponentInfo`` in python
+
+Print indices of detectors in "bank1" that are masked
+
+.. code-block:: python 
+
+  from mantid.simpleapi import CreateSampleWorkspace
+  from mantid.geometry import ComponentInfo, ComponentInfoPythonIterator
+  from mantid.geometry import DetectorInfo, DetectorInfoPythonIterator
```

Comment:
Are these imports really required, even if we are not instatiating tthose objects by hand?

Reply:
Yes, can remove those. Will do.